Source/WebCore/style/values/primitives/StyleZoomNeededToken.h

```diff
@@ -30,5 +30,14 @@ namespace Style {
 // Token passed around to indicate that the evaluation will need zoom passed in the future.
 struct ZoomNeeded { };
 
+struct ZoomFactor {
```

Comment:
added a dedicated type for zoom values instead of using floats.

when `R` has no zoom bit, `ZoomNeeded {}` is passed. if it does have a zoom bit, then `ZoomFactor` must be passed.

callers in RenderStyle must use `usedZoomForLength` when zoom is required.

Reply:
Either the file should be renamed, or this should go in its own file.